_i18n/es/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,114 @@
 
     integrated_address 12346780abcdef00
 
-Payments made to an integrated address generated from your account will go to your account,
-with that payment id attached, so you can tell payments apart.
-
+Pagos realizados a una dirección integrada generada desde tu cuenta irán a tu cuenta, con el ID de pago adjunto, así que podrás diferenciar pagos.
 
-## Proving to a third party you paid someone
+## Probando a un tercero que pagaste a alguien
 
-If you pay a merchant, and the merchant claims to not have received the funds, you may need
-to prove to a third party you did send the funds - or even to the merchant, if it is a honest
-mistake. Monero is private, so you can't just point to your transaction in the blockchain,
-as you can't tell who sent it, and who received it. However, by supplying the per-transaction
-private key to a party, that party can tell whether that transaction sent monero to that
-particular address. Note that storing these per-transaction keys is disabled by default, and
-you will have to enable it before sending, if you think you may need it:
+Si pagas a un comerciante, y el comerciante reclama que no ha recibido el pago, puedes necesitar probar a un tercero que sí enviaste los fondos, o incluso al comerciante, si es que es un error honesto. Monero es privado, así que no puedes simplemente indicar tu transacción en la blockchain, tampoco puedes saber quién la envió, ni quién la recibió. No obstante, proveyendo la llave privada por transacción a un equipo, ese equipo puede saber si esa transacción envió Monero a esa dirección en particular. Ten en cuenta que guardar estas llaves privadas por transacción está desactivado por defecto, y tendrás que activarlo antes de enviar, si crees que lo puedes necesitar:
 
     set store-tx-info 1
 
-You can retrieve the tx key from an earlier transaction:
+Puedes recuperar la llave tx de una transacción anterior:
 
     get_tx_key 1234567890123456789012345678901212345678901234567890123456789012
 
-Pass in the transaction ID you want the key for. Remember that a payment might have been
-split in more than one transaction, so you may need several keys. You can then send that key,
-or these keys, to whoever you want to provide proof of your transaction, along with the
-transaction id and the address you sent to. Note that this third party, if knowing your
-own address, will be able to see how much change was returned to you as well.
+Coloca el ID de la transacción del cual quieres la llave. Recuerda que un pago debe haber sido dividido en más de una transacción, así que puedes necesitar varias llaves. Puedes enviar esa llave, o llaves, a quien quieras proveer con pruebas de tu transacción, junto con los ID de transacción y la dirección a la que enviaste. Ten en cuenta que este tercero, si conoce tu dirección, será capaz de ver cuánto cambio regresó a tu cuenta también.
```

Comment:
Yes, you're right. I occasionally have doubts about all that words because in a normal conversation you can use the one you like more (should, could, may, might) xD.

Reply:
it's ok, I know this takes time and work, you have sure mage a lot of work on this.

Reply:
Yes, and with other stuff to do and that last sickness I got things got slower, really, I haven't seen so many comments on a PR xD

Reply:
This might be of the all files compiled in one commit.